lib/index.js

```diff
@@ -72,7 +72,7 @@ function addRoutes(app) {
 
       // add public key
       brKey.addPublicKey(
-        req.user.identity, publicKey, privateKey, function(err) {
+        {actor: req.user.identity, publicKey, privateKey}, err => {
```

Comment:
All of these will need to change to:

```{actor: req.user.actor, ...```

Once https://github.com/digitalbazaar/bedrock-passport/pull/24 is merged.